tests/integration/targets/utils_cli_parse/output/ios_show_version_parsed.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
I'm not sure I understand how the below parser would result in this data.

Reply:
it parses the version in online and offline mode when we execute `show version` command
![image](https://github.com/user-attachments/assets/c5ab320f-c102-46f7-973b-b313e08d377f)